src/function_file.py

```diff
@@ -22,19 +23,28 @@ def _write_files_to_zip_buffer(zf: ZipFile, directory: Path):
             zf.write(Path(dirpath) / f)
 
 
+@retry(exceptions=FunctionDeployError, tries=12, delay=2, jitter=2, max_delay=15, logger=None)
```

Comment:
holly ...!

Reply:
Haha, just waiting for a file being uploaded 😉